package.json

```diff
@@ -833,6 +833,11 @@
           "default": "cuk",
           "description": "Compilation flags."
         },
+        "objectscript.searchAllDocTypes": {
+          "type": "boolean",
+          "default": false,
+          "description": "Whether server side search should search across all Studio Document types. Default is to only search classes, routines and include files."
```

Comment:
I suggest making this a `markdownDescription` so it can use the term "Quick Open" instead of "server side search", and can hyperlink "Quick Open" to https://code.visualstudio.com/docs/getstarted/tips-and-tricks#_quick-open

Reply:
Ah thanks! I did not know what the correct term was. Will make the update